src/Tickets/Commerce/Reports/Data/Order_Summary.php

```diff
@@ -0,0 +1,265 @@
+<?php
+namespace TEC\Tickets\Commerce\Reports\Data;
+
+use TEC\Tickets\Commerce\Status\Completed;
+use TEC\Tickets\Commerce\Status\Status_Handler;
+use TEC\Tickets\Commerce\Ticket;
+use TEC\Tickets\Commerce\Utils\Value;
+use Tribe__Tickets__Ticket_Object as Ticket_Object;
+use Tribe__Tickets__Tickets;
+
+/**
+ * Class Order_Summary.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Reports\Data
+ */
+class Order_Summary {
+	/**
+	 * @var int The post ID.
```

Comment:
```suggestion
         * @since TBD
         *
	 * @var int The post ID.
```

Here and for all other properties, add the `@since TBD` entry.